style: apply /style-guide pass to sandboxes (#2668)
wandb/docs: sandboxes/create-sandbox.mdx

---
title: Create sandboxes
description: Learn how to create Serverless Sandboxes.

> keywords: ["Sandbox.run", "SandboxDefaults", "sandbox session", "container image"]

Reviewer: ```suggestion
keywords: ["Sandbox.run", "SandboxDefaults", "Session"]
```

Author: Applied in 1159ffaa8.

---

<Warning>
Serverless Sandboxes is in public preview.
</Warning>

> This page shows you how to create Serverless Sandboxes with W&B. You learn how to start a single sandbox, run a main command inside it, and manage multiple sandboxes with a session. Each sandbox runs in its own container with its own [filesystem](/sandboxes/file-access), network, and process space, which gives you an isolated environment to run code.

Reviewer: Style questions

A) I was taught to avoid using "This page shows you..." 
B) Also was taught to use action-oriented phrasing.

```suggestion
Create W&B Serverless Sandboxes to run code in isolated environments. Start a single sandbox, run commands in it, and manage multiple sandboxes with a session. Each sandbox runs in its own container, with its own [[file system](https://chatgpt.com/sandboxes/file-access)](/sandboxes/file-access), network, and process space.
```

Author: Applied in 1159ffaa8. Kept your action-oriented phrasing; I also fixed the file-access link, which had a stray chatgpt.com URL from the paste.
